src/DatePicker/DatePicker.js

```diff
@@ -477,6 +479,8 @@ DatePicker.propTypes = {
         /** Text of the validation message */
         text: PropTypes.string
     }),
+    /** Number to indicate which day the week should start. 1 = Sunday, 2 = Monday, 3 = Tuesday, 4 = Wednesday, 5 = Thursday, 6 = Friday, 7 = Saturday */
+    weekdayStart: PropTypes.number,
```

Comment:
Getting the prop up through the DatePicker

Reply:
@jbadan it seems Calendar.basePropTypes was removed in https://github.com/SAP/fundamental-react/pull/1010/files#diff-35d7f79fc5e005a5d077c11beac4ece9L674 . Is it better to duplicate shared Calendar propTypes over to the DatePicker, or should it even be defined in DatePicker's propTypes in the first place?

Reply:
kind of related: Would we want a PR in https://github.concur.com/coreui/nui-widgets-lab/blob/master/src/DatePicker/DatePicker.jsx as well to add the `weekdayStart` to the PropTypes. It looks like it would work as is because we spread `...otherProps` but not sure if we would want to be explicit there. I don't know if that decision would influence the decision here.

Reply:
@jbadan ?

Reply:
Sorry, I missed this notification. Can we use `CustomPropTypes.range` here? https://github.com/SAP/fundamental-react/blob/master/src/utils/CustomPropTypes/CustomPropTypes.js#L42

`Calendar.basePropTypes` was removed because it did not play nicely with the automated prop table creation. https://github.com/SAP/fundamental-react/blob/master/src/DatePicker/DatePicker.js#L439 is a typo and should be `...Calendar.propTypes`. Then we won't need to duplicate everything and the sharing should work again. Good catch!